pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -288,7 +294,12 @@ func (p *AnacondaInstallerISOTree) serialize() osbuild.Pipeline {
 		// If the KSPath is set, we need to add the kickstart stage to this (bootiso-tree) pipeline.
 		// If it's not specified here, it should have been added to the InteractiveDefaults in the anaconda-tree.
 		if p.KSPath != "" {
-			kickstartOptions, err := osbuild.NewKickstartStageOptions(p.KSPath, makeISORootPath(p.PayloadPath), p.Users, p.Groups, "", "", p.OSName)
+			kickstartOptions, err := osbuild.NewKickstartStageOptionsWithLiveIMG(
+				p.KSPath,
+				p.Users,
+				p.Groups,
+				makeISORootPath(p.PayloadPath))
+
```

Comment:
Note to self: `p.OSName` disappeared here as an argument as it isn't used with live img.